service-request-infno-spec-1.0.0-swagger.json

```diff
@@ -0,0 +1,213 @@
+{
+  "openapi" : "3.0.0",
+  "servers" : [ {
+    "url" : "https://qaeservices1.capetown.gov.za/coct/api"
```

Comment:
remove for now - we'll add these when I've reserved a domain for this

Reply:
Removed